### PR TITLE
Widgets - Closes#4

### LINKED ISSUE #4: Update Widgets Folder
Adding files for stateful and stateless widgets

### PR DESCRIPTION
**Pull request by 27vinayvyas**
Resolves #4 
Widget (stateless and stateful) classes added.

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -4,6 +4,8 @@ import 'package:provider/provider.dart';
 import './shared/providers/name_notifier.dart';
 import './routes.dart';
 import './modules/auth/auth_screen.dart';
+import './shared/models/index.dart';
+import './shared/widgets/index.dart';
```
Comment (yashwp) [inline lib/main.dart:8]: @27vinayvyas No need to import files in main.

File: lib/shared/models/model_name.dart [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class ModelName {
+  int id;
+  var field_name;
+
+  ModelName(this.id, this.field_name);
+}
```
Comment (yashwp) [inline lib/shared/models/model_name.dart:6]: @27vinayvyas No need to create a dummy files. `index.dart` should be enough.

File: lib/shared/widgets/index.dart [hunk 3]
```diff
@@ -0,0 +1,2 @@
+import './stateful_widget.dart';
+import './stateless_widget.dart';
```
Comment (yashwp) [inline lib/shared/widgets/index.dart:2]: @27vinayvyas Same no need to create these files. Delete them.

Comment (27vinayvyas) [reply lib/shared/widgets/index.dart:2]: @yashwp The widgets and models(#4 and #5) issued are not useful then.

Comment (yashwp) [reply lib/shared/widgets/index.dart:2]: Yeah... You can remove them.

Comment (yashwp) [reply lib/shared/widgets/index.dart:2]: @27vinayvyas

File: lib/shared/models/model_name.dart (hunk 2 above)
Comment (yashwp) [reply lib/shared/models/model_name.dart:6]: @27vinayvyas I think you missed this one.

**Pull request closed without merging**